app.js

```diff
@@ -231,6 +232,22 @@ app.on('ready', () => {
     mainWindow = null
   })
 
+  //display config
+  const handleScreenStatusChange = () => {
+    mainWindow.webContents.send('screen-status-change', screen.getAllDisplays())
+  }
+  ipcMain.on('get-all-displays', (e) => {
```

Comment:
for ipc calls I think it is better that we start to use scoped event names like `display::remove-all`/`display/remove-all